floodviz/tests/test_utils.py

```diff
@@ -0,0 +1,210 @@
+import unittest
+import requests_mock
+from floodviz.utils import parse_rdb
+
+
+class TestRdbParsePeak(unittest.TestCase):
```

Comment:
With one test case class, should I include the tests for the webservice that the map_utils currently uses (TestRdbParseMap)? Or should I remove the test for daily value?

I did already remove a good portion of the response, but I will remove more.